src/components/autozoomControl.ts

```diff
@@ -93,11 +97,15 @@ export default class AutozoomControl implements ICnnControl {
   }
 
   private _validateOptions(options: AutozoomControlOpts) {
-    if (
-      options.shouldAutoFrame === false &&
-      options.mode &&
-      options.mode !== AutozoomModes.NORMAL
-    ) {
+    if (!options.mode || !Object.values(AutozoomModes).includes(options.mode)) {
```

Comment:
Do we need to check `!options.mode`, isn't it enough to just do the includes check?

Reply:
Good point